.travis.yml

```
@@ -84,7 +84,7 @@ jobs:
 
     - stage: Lint
       python: 3.6
-      script: isort --check-only -rc yt/
+      script: isort --check-only -rc 
```

while this may be correct (not 100% sure), it think this would be clearer
```suggestion
      script: isort --check-only -rc .
```